### DISCUSSION TITLE
Recurring Transactions & Bills

### DISCUSSION THREAD

**Discussion by neilcrew** [firefly-iii/firefly-iii#5881] [Firefly III] [answered]
What are the specifics of why I might use Bills or Recurring Transactions for my monthly payments (rent, utilities, subscriptions)?

**Comment by JC5** [comment, marked as answer]
The documentation is not enough?

**Reply by neilcrew** [reply]
Hadn't found the detail after an initial scan of the documentation but have now - many thanks.
